v22.1/architecture/overview.md

```diff
@@ -38,24 +38,7 @@ With the confluence of these features, we hope that CockroachDB helps you build
 
 ## Glossary
 
```

Comment:
I think this is a net UX loss from what this page had before.  The new glossary page has a large list of terms that are not necessarily relevant to someone who wants to read specifically about the architecture for learning.

 Speaking as a user, it's just plain harder to go read some other tab when I used to have the info on this page.  

Any chance we can use an include here to add back these terms and concepts, while still also sharing them into the new thing?